crates/task/src/task_template.rs

```diff
@@ -61,7 +61,9 @@ pub struct TaskTemplate {
     /// If this task should start a debugger or not
     #[serde(default, skip)]
     pub task_type: TaskType,
-    /// Represents the tags which this template attaches to. Adding this removes this task from other UI.
+    /// Represents the tags which this template attaches to.
+    /// Adding this removes this task from other UI and gives you ability to run it by tag..
```

Comment:
```suggestion
    /// Adding this removes this task from other UI and gives you ability to run it by tag.
```

This is not very clear to me: we do show these in the tasks modal (as our recent long tags design thread shows), and the gutter test runners are unaffected by custom tasks.

So, which "other UI" starts to lack these? If none, let's remove the first part of the phrase.

Reply:
I also not sure what these other UI are, decided to leave it just in case 😅